CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.9.0 (2020-03-20)
+
+- Fixed the datepicker component to not error when invalid numeric dates are entered. [#127](https://github.com/blackbaud/skyux-datetime/pull/127)
+- Fixed the fuzzy date picker component to not clear invalid dates from the field on blur. [#122](https://github.com/blackbaud/skyux-datetime/pull/122)
```

Comment:
Fixed the fuzzy date picker component to keep invalid dates from the field on blur.

Does that above change the meaning? Trying to write around "to not"

Reply:
`datepicker` should be one word.

Also, it might just be me, but changing `not clear` to `keep` seems unclear. What do you think of changing `to not clear` to `to no longer clear`? Or as an alternative, `to prevent it from clearing`?

Also, do we need `from the field`, @Blackbaud-TrevorBurch? I'd delete it if not. ... And we say `on blur`, but looking at the issue, the description sounds like this happens when the field loses focus. Would it be clearer to change `on blur` to `when the datepicker loss focus`? Seems like it to me, but I'm not sure.

Reply:
Ok. I think I crafted an entry based on all of this feedback.